src/components/Spinner/index.tsx

```diff
@@ -0,0 +1,106 @@
+import * as React from 'react';
+import styled, { keyframes } from 'styled-components';
+import { DefaultProps, RenditionSystemProps } from '../../common-types';
+
+import asRendition from '../../asRendition';
+import { px } from '../../utils';
+import { Flex } from '../Flex';
+import Txt from '../Txt';
+
+const rotate360 = keyframes`
```

Comment:
Is there a common file for 'standard' definitions like this?

Reply:
@grahammcculloch not yet, we haven't put much focus on animations so far. I will be cleaning up rendition in the following months, so some of the common parts will be refactored (not necessary yet).